src/docs/get-started/codelab.md

```diff
@@ -223,7 +223,7 @@ as well as many other open source packages, on [pub.dev][].
        flutter:
          sdk: flutter
        cupertino_icons: ^1.0.2
-    +  english_words: ^4.0.0
+    +  english_words: ^4.0.0-0
```

Comment:
For some reason this is changed by the `./tool/refresh-code-excerpts.sh` script, even if I didn't change the original file

Reply:
Yes, this is from a PR that @domesticmouse created and I merged and it broke the build. I'm not quite sure why it broke the build, as 4.0.0 *is* the latest release for english_words. But, in any case, it's nothing to do with your PR.  :)

Reply:
The issue is that I didn't use `./tool/refresh-code-excerpts.sh` to update this file. I just updated it directly.

Please the original file to have `english_words: ^4.0.0` as the dep constraint